go/deploy-cli/cmd/certManager/secrets.go

```diff
@@ -0,0 +1,68 @@
+package certManager
+
+import (
+	"os"
+
+	"github.com/rackerlabs/understack/go/deploy-cli/cmd"
+	"github.com/rackerlabs/understack/go/deploy-cli/helpers"
+
+	"github.com/charmbracelet/log"
+	"github.com/gookit/goutil/fsutil"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	cmd.RootCmd.AddCommand(CertManager)
+}
+
+var CertManager = &cobra.Command{
+	Use:   "certmanager-secrets",
+	Short: "Generate certmanager-secrets secrets",
+	Long:  "",
+	Run:   certManagerGen,
+}
+
+func certManagerGen(cmd *cobra.Command, args []string) {
+	clusterIssuer()
+}
+
+var clusterIssuerTemplate = `---
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: understack-cluster-issuer
+  annotations:
+    argocd.argoproj.io/sync-wave: "5"
+spec:
+  acme:
+    email: undercloud-dev+uc-iad3-dev@rackspace.com
+    privateKeySecretRef:
+      name: letsencrypt-prod
+    server: https://acme-v02.api.letsencrypt.org/directory
+    solvers:
+      - http01:
+          ingress:
+            ingressClassName: nginx
+        selector:
+          matchLabels:
+            authorizeWith: http
+      - dns01:
+          webhook:
+            groupName: acme.undercloud.rackspace.net
+            solverName: rackspace
+            config:
+              authSecretRef: cert-manager-webhook-rackspace-creds
+              domainName: dev.undercloud.rackspace.net
```

Comment:
So there's a few hardcoded values like this. I was also wondering if we can have the templates external and then compiled in so we can lint them somehow?